modules/UI/authentication/LoginDialog.js

```diff
@@ -94,7 +94,8 @@ function LoginDialog(successCallback, cancelCallback) {
                     }
                 } else {
                     // User cancelled
-                    cancelCallback();
```

Comment:
I'd like to set default values for callbacks like:

`function LoginDialog(successCb = $.noop, cancelCb = $.noop)`

instead of checking for the type of argument.